ostelco-ios-client/Coordinators/OnboardingCoordinator.swift

```diff
@@ -377,23 +377,49 @@ extension OnboardingCoordinator: SingPassCoordinatorDelegate {
 }
 
 extension OnboardingCoordinator: MyInfoSummaryDelegate {
+    func fetchMyInfoDetails(_ controller: MyInfoSummaryViewController, code: String, completion: @escaping (MyInfoDetails) -> Void) {
+        let spinnerView = controller.showSpinner(loadingText: NSLocalizedString("Loading your data from SingPass...", comment: "Loading text after user approves SingPass"))
+        primeAPI.loadSingpassInfo(code: code)
+        .ensure {
+            controller.removeSpinner(spinnerView)
+        }
+        .done { myInfoDetails in
+            completion(myInfoDetails)
+        }
+        .catch { error in
+            ApplicationErrors.log(error)
```

Comment:
This changes the way we handle errors. Last implementation will 
1) Show error
2) Go back to Select verification method screen.